packages/@aws-cdk/cfnspec/spec-source/000_sam.spec.json

```diff
@@ -1,5 +1,22 @@
 {
     "PropertyTypes": {
+        "AWS::Serverless::Application.ApplicationLocation": {
```

Comment:
Should I be writing this directly into the spec or a patch file? E.g. why is deployment preferences a patch and not in this spec file? @jlhood

Reply:
This is my first time looking at these files, so I don't think I'm the person who can answer this question... I'm not sure what the organization model is for these files.

Reply:
We talked about trying to get them updated at the source GoFormation which I believe @PaulMaddox owns.
Until then, you should create a patch file for this and _not_ update the spec directly because it will be overwritten as soon as someone runs our update scripts.